src/pages/EditReportFieldDropdownPage.tsx

```diff
@@ -92,6 +92,7 @@ function EditReportFieldDropdownPage({fieldName, onSubmit, fieldID, fieldValue,
                         textInputLabel={translate('common.search')}
                         boldStyle
                         sections={sections}
+                        focusedIndex={0}
```

Comment:
does this have to be set to 0?

Reply:
Can you put that in a comment to make this very clear?

Reply:
Please explain the _why_ in the comment